### PR TITLE
RD-307 Transfer learning potato

### PR DESCRIPTION
**Pull request by melissande**
This PR is quite big as it does implement the adaptation of Mask R CNN for potato semantic segmentation. 
Many documentation resources are available:
1) [Principle of the Mask R CNN algorithm](https://hummingbirdtech.atlassian.net/wiki/spaces/HT/pages/597295668/1.+Mask+R-CNN+-+Paper+description+and+adaptation)
2) [Dataset preparation](https://hummingbirdtech.atlassian.net/wiki/spaces/HT/pages/600604673/2.+Mask+R-CNN+-+DataSet+Preparation)
3) The [README.md](https://github.com/HummingbirdTechGroup/Mask_RCNN/blob/RD-307/transfer-learning-potato/README.md) also explains very well what's in the  repository and the main code is located `mrcnn/model.py` and it is explained in the videos located in this [drive](https://drive.google.com/drive/folders/1OZhSTLq3Hc_HGYnGvoy4KHk59eB-X0C7?usp=sharing)

There is no emergency in re-reading this PR as it is still a proof of concept and should not go in production.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,4 +151,111 @@ id_projected_cell_boundary: int
...
+NAME = "potato"
+
+# Train on 1 GPU and 8 images per GPU. We can put multiple images on each
+# GPU because the images are small. Batch size is 8 (GPUs * images/GPU).
+GPU_COUNT = 1
+IMAGES_PER_GPU = 8
+
+
+ # Number of classes (including background)
+NUM_CLASSES = 1 + 1  # background + 3 shapes
```
[the remark this answers is not in the document]
Comment (melissande) [reply README.md]: will correct it

**Comment by melissande** [comment]
> Under retrain potato you mention a training made on simple shape such as triangles, rectangles, etc...
> 
> May need to update the markers there.

yes I will do a clean up because I noticed other documentation mistaktes

**Comment by VirginieBfd** [comment]
@melissande should I merge or should I not?

https://www.youtube.com/watch?v=BN1WwnEDWAM

**Comment by melissande** [comment]
> @melissande should I merge or should I not?
> 
> https://www.youtube.com/watch?v=BN1WwnEDWAM

hhaah don't ! I am waiting for the new dataset POTATO_V20 from CF!

**Comment by dataland** [comment]
remove me from this thread

**Pull request merged**